### DISCUSSION TITLE
Blazor with CoreWCF?

### DISCUSSION THREAD

**Discussion by Grauenwolf** [CoreWCF/CoreWCF#1300] [General]
Are there any examples of using CoreWCF as the communication layer for WASM Blazor?

I ask because the standard approach of using HttpClient is a right pain. I feel this would be a great place to highlight CoreWCF... if it works with WASM.

**Comment by g7ed6e** [comment]
@Grauenwolf client side WCF support in Blazor WASM is tracked here https://github.com/dotnet/wcf/issues/5335
